Makefile

```diff
@@ -5,20 +5,36 @@ BROKER_IMAGE     ?= $(REGISTRY)/$(ORG)/ansible-service-broker
 BUILD_DIR        = "${GOPATH}/src/github.com/openshift/ansible-service-broker/build"
 PREFIX           ?= /usr/local
 BROKER_CONFIG    ?= $(PWD)/etc/generated_local_development.yaml
+SOURCE_DIRS      = cmd pkg
 SOURCES          := $(shell find . -name '*.go' -not -path "*/vendor/*")
 SVC_ACCT_DIR     := /var/run/secrets/kubernetes.io/serviceaccount
 KUBERNETES_FILES := $(addprefix $(SVC_ACCT_DIR)/,ca.crt token tls.crt tls.key)
 .DEFAULT_GOAL    := build
 
-vendor:
+vendor: ## Install project dependencies
 	@glide install -v
 
-broker: $(SOURCES)
-	go build -i -ldflags="-s -w" ./cmd/broker
+broker: $(SOURCES) ## Build the broker
+	@go build -i -ldflags="-s -w" ./cmd/broker
 
-build: broker
+build: broker ## Build the project
 	@echo > /dev/null
 
+lint: ## Run golint
+	@golint -set_exit_status $(SOURCE_DIRS)
+
+fmt: ## Run go fmt
+	@gofmt -l -w $(SOURCE_DIRS)
+
+fmtcheck: ## Check go formatting
+	@gofmt -l $(SOURCE_DIRS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
```

Comment:
IMO seeing the diff would be nice. To be honest, I almost never see this or use this because my editor is configured to do it for me, so I don't have a strong opinion.